.env.default

```diff
@@ -1,19 +1,25 @@
-DATABASE_URL = "sqlite:///./test.db"
+### Indexer ###
+# Graph DB
+GRAPH_DB_BASE_URL = "http://localhost:3030/resources"
+GRAPH_DB_USERNAME = "admin"
+GRAPH_DB_PASSWORD = "admin"
+REGEN_NODE_TENDERMINT_WS_URL = "ws://localhost:26657/websocket"
+
+### Pinning service ###
+# Postgres DB
 POSTGRES_SERVER = "localhost"
 POSTGRES_USER = "postgres"
 POSTGRES_PASSWORD = "postgres"
 POSTGRES_DB = "app"
 
+# Regen node
 REGEN_NODE_REST_URL = "http://localhost:1317"
-GRAPH_DB_BASE_URL = "http://localhost:3030/resources"
-GRAPH_DB_USERNAME = "admin"
-GRAPH_DB_PASSWORD = "admin"
 REGEN_NODE_TENDERMINT_RPC_URL = "http://localhost:26657"
-REGEN_NODE_TENDERMINT_WS_URL = "ws://localhost:26657/websocket"
-REGEN_CHAIN_ID = "chain-YsT5fv"
+
+# TX Signing
+REGEN_RESOLVER_ID = 1
+REGEN_CLI_COMMAND = "docker run --rm -i --network host -v $(pwd)/localnet/keys:/regen-keys ghcr.io/cambiumteam/regen-cli:latest"
```

Comment:
@wgardiner I updated the `REGEN_CLI_COMMAND` to use the `localnet/keys` directory for the docker volume.